src/tests/ftest/util/container_rf_test_base.py

```diff
@@ -186,7 +186,7 @@ def execute_cont_rf_test(self, create_container=True):
             # Refresh local pool and container
             self.log.info("==>(6)Check for pool and container info after rebuild.")
             self.pool.check_pool_info()
-            self.container.check_container_info()
+            self.container.query()
```

Comment:
This is because previously, `check_container_info()` without arguments wasn't verifying anything. It was just calling container query through the API and not doing anything with the return